<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiddlewarePracticaFinalClass</class>
 <widget class="QMainWindow" name="MiddlewarePracticaFinalClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiddlewarePracticaFinal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="placeholder" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>681</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 224, 255);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_rot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotation:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_pos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>13</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sca">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Scale:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_rot">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>42</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x_sca">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>73</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xPosition">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xRotation">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_pos">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>13</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yScale">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yPosition">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="yRotation">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_z_sca">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>64</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z_pos">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>13</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zScale">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zRotation">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="zPosition">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
     <property name="value">
      <double>-2.750000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_rot">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>42</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z_rot">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>42</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y_sca">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>73</y>
       <width>7</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="xScale">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_rotation_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string>Render</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotation_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(185, 185, 185);</string>
    </property>
    <property name="text">
     <string>Hierarchy</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>215</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(214, 214, 214);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listObjects"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/MiddlewarePracticaFinal.qrc"/>
 </resources>
 <connections/>
</ui>
